src/libraries/System.Private.DataContractSerialization/src/System/Xml/XmlBaseReader.cs

```diff
@@ -590,10 +585,8 @@ private XmlAttributeNode GetAttributeNode(int index)
             return null;
         }
 
-        private XmlAttributeNode? GetAttributeNode(string localName, string? namespaceUri)
+        private XmlAttributeNode? GetAttributeNode(string localName!!, string? namespaceUri)
         {
-            if (localName == null)
-                throw System.Runtime.Serialization.DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentNullException(nameof(localName)));
             if (namespaceUri == null)
```

Comment:
I think at one point it may have also contained tracing?  But now that it's just:
https://github.com/dotnet/runtime/blob/2b862d43469a79368fa20413d44e3943524daa0b/src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DiagnosticUtility.cs#L47-L50
I have no qualms bypassing it :)